lambdas/postgres-migration-async-operation/src/index.ts

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const asyncOperations = require('@cumulus/async-operations');
+const { models } = require('@cumulus/api');
+const Logger = require('@cumulus/logger');
+
+const logger = new Logger({ sender: '@cumulus/postgres-migration-async-operation' });
+exports.handler = async () => {
+  const stackName = process.env.stackName;
```

Comment:
If the env variables `stackName` and `system_bucket` aren't explicitly used by a dependency, consider defining them as the keys in the invocation of `startAsyncOperation`

Reply:
[`ff56663` (#2165)](https://github.com/nasa/cumulus/pull/2165/commits/ff56663939c03cb4e00e6e67dc2ba42ca30d5ed6)